docs/HowToUse.md

```diff
@@ -8,6 +8,14 @@ First, you need the ONE toolchain. Let's install the ONE toolchain. (You should
 
 <img src="https://user-images.githubusercontent.com/7223627/174947559-71ab213c-68c4-4700-bcec-e55f6a2203bf.gif" width=600 />
 
+## Set default toolchain
+
+To compile model using ONE toolchain, you must specify the default toolchain. To specify a default toolchain, if you mouse up the toolchain item you want to use in Toolchain view, :heavy_check_mark: icon appears. Click this icon to specify it as default toolchain. If toolchain is specified as default toolchain, the icon changes.
+
+| Toolchain | Default Toolchain |
+|-----------|-------------------|
+| <img src="https://user-images.githubusercontent.com/7223627/177962615-71e64f52-e684-432d-a053-07896a20a479.png" width=300 /> | <img src="https://user-images.githubusercontent.com/7223627/177963453-d22998a0-be21-4531-a70b-9057a204e7eb.png" width=300 /> |
+
 ## Prepare a NN model and a config file
 
 Second, you need a NN model and a config file to run compiling. Now you can get one from [res/modelDir/truediv](../res/modelDir/truediv/). Add it to your workspace. Now you can see a model and a config.
```

Comment:
We may not need `First,` and `Second,` at L7 and L21. Or we could add `second,` at L13.